std_msgs/Int16 redPoints
std_msgs/Int16 bluePoints
string zoneName
string[] modelNames
